train.py

```diff
@@ -168,18 +182,22 @@ def on_validation_end(self, trainer, pl_module):
 
     # TRAINING PARAMETERS
 
-    parser.add_argument('--epochs', dest='epochs', type=int, default=40)
+    parser.add_argument('--epochs', dest='epochs', type=int, default=20)
     parser.add_argument('--patience', dest='patience', type=int, default=10)
-    parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
-    parser.add_argument('--lr', dest='l_rate', type=float, default=1e-4)
-    parser.add_argument("--min-lr", dest='minimum_lr', type=float, default=1e-5, help="Minimum Learning Rate")
-    parser.add_argument("--lr-decay", dest='lr_decay', type=float, default=1e-3, help="Learning rate (weight) decay")
+    #batch_size 1 for graph networks
+    # parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
+    parser.add_argument('--batch-size', dest='batch_size', type=int, default=1)
+    #higher learning rate for graph models
+    # parser.add_argument('--lr', dest='l_rate', type=float, default=1e-4)
+    parser.add_argument('--lr', dest='l_rate', type=float, default=0.01)
+    parser.add_argument("--min-lr", dest='minimum_lr', type=float, default=0.001, help="Minimum Learning Rate")
+    parser.add_argument("--lr-decay", dest='lr_decay', type=float, default=0.01, help="Learning rate (weight) decay")
 
     # CONFIGURATION
 
-    parser.add_argument('--dataset', dest='dataset', default='R8Text', choices=SUPPORTED_DATASETS,
+    parser.add_argument('--dataset', dest='dataset', default='R8Graph', choices=SUPPORTED_DATASETS,
```

Comment:
R8Graph is not a supported dataset in SUPPORTED_DATASETS

Reply:
I added it (and also R52)